src/main/java/com/gachigang/ontherun/config/handler/ExceptionWebHandler.java

```diff
@@ -0,0 +1,32 @@
+package com.gachigang.ontherun.config.handler;
+
+import com.gachigang.ontherun.model.dto.ErrorMessage;
+import jakarta.annotation.Nonnull;
+import lombok.NonNull;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@Component
+public class ExceptionWebHandler {
+
+    public HttpStatus getHttpStatus(@Nonnull final Exception exception,
+                                    @Nonnull final HttpStatus fallBackHttpStatus) {
+        final ResponseStatus responseStatus = exception.getClass().getAnnotation(ResponseStatus.class);
+        return responseStatus != null ? responseStatus.value() : fallBackHttpStatus;
```

Comment:
fallBackHttpStatus -- not clear name, what it the reason for calling it like that?

Reply:
Oh, I guess, it would be better with the name like status.